### ISSUE TITLE
ChildProcessError: [Errno 10] No child processes

### ISSUE THREAD

**Issue by Nagutos** [smacke/ffsubsync#172]
**Environment :**
 - OS: Windows 10
 - python version (`python 3.9`)
 - subsync version (`subsync 0.4.22`)

Hello I have this error when I start using ffs for this commande : ffsubsync E1.mkv -i S1.srt -o S1V2.srt

```
Traceback (most recent call last):
  File "C:\Users\Utilisateur\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,"__main__", mod_spec)
  File "C:\Users\Utilisateur\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\Utilisateur\AppData\Local\Programs\Python\Python39\Scripts\ffs.exe\__main__.py", line 7, in <module>
  File "C:\Users\Utilisateur\AppData\Local\Programs\Python\Python39\lib\site-packages\ffsubsync\ffsubsync.py", line 713, in main
    return run(parser)["retval"]
  File "C:\Users\Utilisateur\AppData\Local\Programs\Python\Python39\lib\site-packages\ffsubsync\ffsubsync.py", line 506, in run
    sync_was_successful = _run_impl(args, result)
  File "C:\Users\Utilisateur\AppData\Local\Programs\Python\Python39\lib\site-packages\ffsubsync\ffsubsync.py", line 442, in _run_impl
    reference_pipe.fit(args.reference)
  File "C:\Users\Utilisateur\AppData\Local\Programs\Python\Python39\lib\site-packages\ffsubsync\sklearn_shim.py", line 225, in fit
    self._final_estimator.fit(Xt, y, **fit_params)
  File "C:\Users\Utilisateur\AppData\Local\Programs\Python\Python39\lib\site-packages\ffsubsync\speech_transformers.py", line 417, in fit
    os.waitpid(process.pid, 0)
ChildProcessError: [Errno 10] No child processes
```
It stop at this step : 

```
INFO     extracting speech segments from reference 'E1.mkv'...                                                                                                                                                   ffsubsync.py:441
INFO     Checking video for subtitles stream...                                                                                                                                                        speech_transformers.py:303
INFO     Video file appears to lack subtitle stream                                                                                                                                                    speech_transformers.py:308
```

All requirements are satisfied : 

```
Requirement already satisfied: ffsubsync in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (0.4.22)
Requirement already satisfied: charset-normalizer in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (3.0.1)
Requirement already satisfied: cchardet in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (2.1.7)
Requirement already satisfied: typing-extensions in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (4.4.0)
Requirement already satisfied: numpy>=1.12.0 in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (1.24.1)
Requirement already satisfied: rich in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (13.1.0)
Requirement already satisfied: future>=0.18.2 in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (0.18.3)
Requirement already satisfied: tqdm in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (4.64.1)
Requirement already satisfied: pysubs2>=1.2.0 in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (1.6.0)
Requirement already satisfied: auditok==0.1.5 in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (0.1.5)
Requirement already satisfied: chardet in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (5.1.0)
Requirement already satisfied: six in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (1.16.0)
Requirement already satisfied: ffmpeg-python in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (0.2.0)
Requirement already satisfied: srt>=3.0.0 in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (3.5.2)
Requirement already satisfied: webrtcvad-wheels in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from ffsubsync) (2.0.11.post1)
Requirement already satisfied: pygments<3.0.0,>=2.6.0 in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from rich->ffsubsync) (2.14.0)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from rich->ffsubsync) (0.9.1)
Requirement already satisfied: colorama in c:\users\utilisateur\appdata\local\programs\python\python39\lib\site-packages (from tqdm->ffsubsync) (0.4.6)
```

**Comment by tonyhsie** [comment]
I've the same issue after upgrading to version 0.4.22.
So I go back to version 0.4.11 and everything is fine.

**Comment by Nagutos** [comment]
> I've the same issue after upgrading to version 0.4.22. So I go back to version 0.4.11 and everything is fine.

Thank you for your help it worked !

**Comment by justusip** [comment]
Please reopen the issue since it is not fixed in the new version

**Comment by smacke** [comment]
Thanks for reporting. This should be fixed in 0.4.23. You can pick it up by running

```
python -m pip install --upgrade ffsubsync
```
